src/coreclr/System.Private.CoreLib/CreateRuntimeRootILLinkDescriptorFile.targets

```diff
@@ -30,7 +26,7 @@
         of child processes, and attach a debugger to msbuild. After that you can set a bp on the exception
         you are seeing.
   -->
-  <UsingTask TaskName="CreateRuntimeRootILLinkDescriptorFile" AssemblyFile="$(_ILLinkTasksPath)" />
+  <UsingTask TaskName="CreateRuntimeRootILLinkDescriptorFile" AssemblyFile="$(ILLinkTasksAssembly)" Condition="'$(ILLinkTasksAssembly)' != ''" />
```

Comment:
For corelib, is there ever a case where `'$(ILLinkTasksAssembly)' == ''`?

Reply:
For any project which hasn't restored yet and included the nuget generated properties and targets files. `ILLinkTasksAssembly` isn't available before that nuget.g.props file is imported. As MSBuild already evaluates UsingTasks before they are used, the `AssemblyFile` parameter must not be empty, hence the condition.